### ISSUE TITLE
How to check the training converges?

### ISSUE THREAD

**Issue by AminSuzani** [lmjohns3/theanets#23]
Hi,

I have a multivariate nonlinear regression problem and I am trying to solve it using deep neural networks. I use the code below for training. My X is 10000_40 and my Y is 10000_78. I was wondering how I can check a few things:
1- How do I know the training converged.
2- How do I know what 'learning rate', 'momentum' and 'update_num' it used as default. 

e = theanets.Experiment(theanets.feedforward.Regressor,
                    layers=(40, 100, 200, 300, 150, 78),
                    optimize='sgd',
                    activation='tanh')  
e.run(train_set, train_set)
Y_predicted = e.network(X_test_minmax)

I tried using 'hf' instead of 'sgd'. It printed some performance variables for each iteration, but it was too slow for my application. The other problem is that when I write 'layerwise' instead of 'sgd', it gives me an error. Any kind of help is appreciated. 

Thanks,
Amin
